docs/man/git-lfs-lock.1.ronn

```diff
@@ -0,0 +1,27 @@
+git-lfs-lock(1) -- Set a file as "locked" on the Git LFS server
+===============================================================
+
+## SYNOPSIS
+
+`git lfs lock` [options] <path>
+
+## DESCRIPTION
+
+Sets the given file path as "locked" against the Git LFS server, with the
+intention of blocking attempts by other users to update the given path. Locking
```

Comment:
I would re-phrase this in terms of what locking a file actually does, not it's intention. In practice, locking a file causes the pre-push command to prevent the push client-side, so we should make that explicit in the command docs.